src/pybind/mgr/dashboard/frontend/src/app/shared/components/modal/modal.component.scss

```diff
@@ -12,6 +12,4 @@
 
 ::ng-deep .modal-body {
   max-height: 70vh;
-  overflow-y: auto;
-  overflow-x: hidden;
```

Comment:
Removing this breaks other modals. 
Like the OSD Flag modal:
![image](https://user-images.githubusercontent.com/399326/85298675-e8f84b80-b493-11ea-8ba0-acd403523e8f.png)

Reply:
Readded them.